### PR TITLE
Some more localized string for Polish

### PR DESCRIPTION
**Pull request by spitfire**

### CODE REVIEW

**Comment by DonSqueak** [comment]
I added some comments to your commits, please check :)

File: CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff [hunk 1]
```diff
@@ -6,23 +6,23 @@
...
         <note>Bundle display name</note>
       </trans-unit>
       <trans-unit id="CFBundleName" xml:space="preserve">
-        <source>Callsheet</source>
+        <source>Callsheet</source><target>Callsheet</target>
         <note>Bundle name</note>
       </trans-unit>
       <trans-unit id="NSLocalNetworkUsageDescription" xml:space="preserve">
-        <source>If Callsheet can use your local network, it may be able to show links to what you're currently watching on the main screen. Enabling local network usage is completely optional. No data about your network is sent off your device.</source>
+        <source>If Callsheet can use your local network, it may be able to show links to what you're currently watching on the main screen. Enabling local network usage is completely optional. No data about your network is sent off your device.</source><target>Jeśli Callsheet może używać Twojej sieci lokalnej, powinno móc pokazać linki to aktualnie odtwarzanych mediów na głównym ekranie. Włączenie dostępu do sieci lokalnej jest opcjonalne. Żadne dane na temat Twojej sieci nie są wysyłane z Twojego urządzenia.</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:17]: ... powinno móc pokazać linki do* aktualnie odtwarzanych mediów na głównym ekranie.

File: CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff [hunk 2]
```diff
@@ -373,96 +373,96 @@
...
         <note/>
       </trans-unit>
       <trans-unit id="Done" xml:space="preserve">
-        <source>Done</source>
+        <source>Done</source><target>Zrobione</target>
         <note/>
       </trans-unit>
       <trans-unit id="Each TV show can have their own settings, set by tapping the `Hide` `Spoilers…` button when looking at a show or episode." xml:space="preserve">
-        <source>Each TV show can have their own settings, set by tapping the `Hide` `Spoilers…` button when looking at a show or episode.</source>
+        <source>Each TV show can have their own settings, set by tapping the `Hide` `Spoilers…` button when looking at a show or episode.</source><target>Każdy serial może mieć swoje własne ustawienia, dostosowywane przez stuknięcie przycisku ‚Ukryj’ ‚Spoilery’ podczas patrzenia na serial lub jego odcinek.</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:416]: Tutaj te ` to są "backticks". Jak chcesz używać "quotes", to spacja nie musi być wykluczona: ‚Ukryj spoilery’ wtedy.

File: CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff [hunk 3]
```diff
@@ -758,51 +758,51 @@
...
         <note>Shown when trying to upsell the user on expensive subscriptions</note>
       </trans-unit>
       <trans-unit id="Or, you can have %lld more free search. Just this once.|==|plural.few" xml:space="preserve">
-        <source>Or, you can have %lld more free searches. Just this once.</source>
+        <source>Or, you can have %lld more free searches. Just this once.</source><target>Możesz mieć %lld dodatkowe darmowe wyszukiwania. Tylko raz.</target>
         <note>Shown when trying to sell more expensive subscriptions</note>
       </trans-unit>
       <trans-unit id="Or, you can have %lld more free search. Just this once.|==|plural.many" xml:space="preserve">
-        <source>Or, you can have %lld more free searches. Just this once.</source>
+        <source>Or, you can have %lld more free searches. Just this once.</source><target>Możesz mieć %lld dodatkowych darmowych wyszukiwań Tylko raz.</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:781]: Brakuje kropka.

File: CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff [hunk 4]
```diff
@@ -1074,159 +1074,159 @@
...
         <note>Subtitle for the App Icon row in Settings</note>
       </trans-unit>
       <trans-unit id="Subscription Status" xml:space="preserve">
-        <source>Subscription Status</source>
+        <source>Subscription Status</source><target>Status subskrypcji</target>
         <note/>
       </trans-unit>
       <trans-unit id="Subscriptions Debugging" xml:space="preserve">
-        <source>Subscriptions Debugging</source>
+        <source>Subscriptions Debugging</source><target>Bedugging subskrypcji</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:1125]: Debugging* ;)

File: CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff [hunk 5]
```diff
@@ -1242,95 +1242,95 @@
...
         <note>Shown to try to encourage the user to rate the app</note>
       </trans-unit>
       <trans-unit id="You can be the first! Don’t be shy!" xml:space="preserve">
-        <source>You can be the first! Don’t be shy!</source>
+        <source>You can be the first! Don’t be shy!</source><target>Możesz być pierwsza/y, nie bądź nieśmiała/y!</target>
         <note>Shown to try to encourage the user to rate the app</note>
       </trans-unit>
       <trans-unit id="You don't seem to have a subscription. 🤨" xml:space="preserve">
-        <source>You don't seem to have a subscription. 🤨</source>
+        <source>You don't seem to have a subscription. 🤨</source><target>NIe wygląda na to, że masz subszkrypcję. 🤨</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:1273]: Nie*

File: CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff [hunk 6]
```diff
@@ -1242,95 +1242,95 @@
...
         <note/>
       </trans-unit>
       <trans-unit id="You have used all your free searches. Subscribe for unlimited searches." xml:space="preserve">
-        <source>You have used all your free searches. Subscribe for unlimited searches.</source>
+        <source>You have used all your free searches. Subscribe for unlimited searches.</source><target>Żużyto wszystkie darmowe wyszukiwania. Subskrybuj, by mieć dostęp do nielimitowanych wyszukiwań.</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:1277]: Zużyto*

File: CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff [hunk 7]
```diff
@@ -1242,95 +1242,95 @@
...
         <note>Header shown for people</note>
       </trans-unit>
       <trans-unit id="Your favorite characters finally get married." xml:space="preserve">
-        <source>Your favorite characters finally get married.</source>
+        <source>Your favorite characters finally get married.</source><target>Twoi ulubieni bohaterowie w końcu pobierają się.</target>
         <note>A fake summary shown when demonstrating hiding spoilers</note>
       </trans-unit>
       <trans-unit id="Your purchase is still pending. Please wait for it to be approved." xml:space="preserve">
-        <source>Your purchase is still pending. Please wait for it to be approved.</source>
+        <source>Your purchase is still pending. Please wait for it to be approved.</source><target>Twój zakup jest przetwarzany. Proszę czekaj aż zostanie zatwierdzony.</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:1289]: przecinek przed aż?

**Comment by DonSqueak** [comment]
Done with another review run. See inline comments above. :)

File: CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff (hunk 7 above)
Comment (spitfire) [reply CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:1289]: Aż musiałem sprawdzić: https://www.interpunkcja.pl/zasady-interpunkcji/przecinek-przed-az.
Poprawiam.

**Comment by spitfire** [comment]
Poprawione

**Review by DonSqueak — APPROVED** [review]

**Comment by cliss** [comment]
I added some new strings; would you mind merging in my changes (from 58c82df0bc8eb5a00080f94b6e64fdfac64a68b4), add them, and then let me know when this is ready to merge! :)

**Review by cliss — CHANGES_REQUESTED** [review]
Please fix merge conflicts and add new strings from 58c82df0bc8eb5a00080f94b6e64fdfac64a68b4 😇

File: CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff [hunk 8]
```diff
@@ -6,23 +6,23 @@
     </header>
     <body>
       <trans-unit id="CFBundleDisplayName" xml:space="preserve">
-        <source>Callsheet</source>
+        <source>Callsheet</source><target>Callsheet</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:9]: BTW, most conflicts get resolve if you replace `</source><target>`

with
```
</source>
<target state="translated">
```

Note the line break.

Comment (DonSqueak) [reply CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:9]: (I guess `state="translated"` is optional, but the "missing" line break confuses gits diff.

Comment (spitfire) [reply CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff:9]: Thanks for the info, I'll see what I can do

**Comment by spitfire** [comment]
> Please fix merge conflicts and add new strings from [58c82df](https://github.com/cliss/callsheet-localizations/commit/58c82df0bc8eb5a00080f94b6e64fdfac64a68b4) 😇

I'll try to, it will be the first time I'll need to fix conflicts in git though :)

**Comment by spitfire** [comment]
@cliss - @DonSqueak has helped me a lot with resolving the conflicts, but the last (very few remain). Can you look at it and advise what I should do?

**Comment by DonSqueak** [comment]
> @cliss - @DonSqueak has helped me a lot with resolving the conflicts, but the last (very few remain). Can you look at it and advise what I should do?

I may have found a solution for this. @spitfire, see latest PR from me --> you, please.

**Comment by spitfire** [comment]
> > @cliss - @DonSqueak has helped me a lot with resolving the conflicts, but the last (very few remain). Can you look at it and advise what I should do?
> 
> I may have found a solution for this. @spitfire, see latest PR from me --> you, please.

Ok, I approved this PR and it shows no conflicts anymore.

**Comment by DonSqueak** [comment]
Then you can have fun with the new strings that @cliss posted yesterday ;)

**Comment by spitfire** [comment]
> Then you can have fun with the new strings that @cliss posted yesterday ;)

Nope, today I'm having "fun" with my 3d printer :D I will get to them, probably tomorrow:)

**Comment by cliss** [comment]
Thanks so much for the assist on this, @DonSqueak. My last 48 hours have been 🌪️  so I apologize for not jumping in myself.

**Review by cliss — APPROVED** [review]

**Pull request merged**